test/helpers/compile.js

```diff
@@ -1,14 +1,15 @@
 const path = require('path');
 const webpack = require('webpack');
+const webpackPackageJson = require('webpack/package.json');
 
 const fixturePath = path.resolve(__dirname, '..', 'fixtures');
 const outputPath = path.resolve(__dirname, '..', 'output');
 
 function compile(fixture, moduleRules, resolveAlias = {}) {
   return new Promise((resolve, reject) => {
     const entry = path.resolve(fixturePath, 'less', `${fixture}.less`);
-
-    webpack({
+    const [majorVersion] = webpackPackageJson.version.split('.');
```

Comment:
Please drop this, I appreciate that you want to fix it, but adding `webpack >= v4.0.0` to the CI setup will be part of #233 asap. The loader works with the `webpack >= v4.0.0` Loader API (no affecting breaking changes between `v3.0.0` => `v4.0.0`)

Reply:
Are you sure? The tests don't pass without this change.

Reply:
Yep, I'm sure about that, please pin `webpack` to `^3.0.0` (https://github.com/webpack-contrib/less-loader/blob/master/package.json#L65)  instead